doc/operator-scope.md

```diff
@@ -83,6 +89,13 @@ With the above changes the specified manifests should look as follows:
       kind: ClusterRole
       name: memcached-operator
       apiGroup: rbac.authorization.k8s.io
+* `deploy/service_account.yaml`
+    ```YAML
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: memcached-operator
+      namespace: <operator-namespace>
     ```
```

Comment:
The operator-SDK te service_account.yaml. See for example https://github.com/operator-framework/operator-sdk-samples/blob/master/go/memcached-operator/deploy/service_account.yaml. 

And then, the namespace is not required on this resource since it will use the namespace where it is applied